alerting/src/main/kotlin/org/opensearch/alerting/transport/TransportGetFindingsAction.kt

```diff
@@ -125,7 +123,8 @@ class TransportGetFindingsSearchAction @Inject constructor(
             queryBuilder.filter(timeRangeQuery)
         }
 
-        if (!detectionType.isNullOrBlank()) {
+        if (!getFindingsRequest.detectionType.isNullOrBlank()) {
```

Comment:
what is detectionType?

Reply:
yes In alerting it has no meannig but we need to filter the SA findings based on the detectionType [threat intel or rule based], this will query on `.opensearch-sap-*-findings` index to do that. I'm open to other suggestions if this is wrong or if we've a better way to do this.

Reply:
pass querybuilder as param in findings request